### PR TITLE
SLA Service added

### PR DESCRIPTION
**Pull request by rosogon**
I also modified ubuntu version to ubuntu/precise. 

I usually have errors deploying sla-db: "could not install curl (locked or without permissions)". It's strange because I manually execute that line in the script and it works. So maybe it's a problem of parallels apt-gets.

Sometimes other services do not install correctly.

### CODE REVIEW

**Comment by rosogon** [comment]
The issues installing mysql on ubuntu14 (libaio not installed) are maybe caused by this problem with apt-get.

File: Vagrantfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ NODE_COUNT = 2 # Number of SeaClouds nodes to start
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Setup VirtualBox VMs
-  config.vm.box = "ubuntu/trusty64"
+  config.vm.box = "hashicorp/precise64"
```
Comment (andreaturli) [inline Vagrantfile]: I'd prefer the official `ubuntu/precise64`

Comment (rosogon) [inline Vagrantfile]: Looked here https://atlas.hashicorp.com/boxes/search. I didn't realize there were more than 1 page. Changing.

**Comment by andreaturli** [comment]
@rosogon I'm testing the new yaml, I can't see the SLA core UI, when clicking on Main URI `<error code="404" message="null for uri: http://192.168.100.11:18080/"/>` Is it known?

**Comment by rosogon** [comment]
I must fix that. Try http://192.168.100.11:18080/providers , please. It should work.

**Comment by andreaturli** [comment]
still no luck, `<providers/>` is the only thing I can see? it that correct?

**Comment by rosogon** [comment]
It's correct. There are no providers in the database.

File: seaclouds.yaml [hunk 2]
```diff
@@ -39,3 +39,24 @@ services:
...
+            DB_USERNAME: "atossla"
+            DB_PASSWORD: "_atossla_"
+      war: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=sla-service&v=LATEST&e=war
+
+
+    - serviceType: brooklyn.entity.database.mysql.MySqlNode
+      id: sla-db
+      name: SLA Db
+      brooklyn.config:
+        creationScriptUrl: https://raw.githubusercontent.com/SeaCloudsEU/sla-core/first_integrated_platform/sla-repository/src/main/resources/sql/01database.sql
```
Comment (andreaturli) [inline seaclouds.yaml:62]: we should find a better way to version that script

**Comment by andreaturli** [comment]
ok @rosogon I think it is ok

let's keep in mind that we may want to extract `sla-core.yaml` into a separate file, so to have a better decoupling.

merging it

File: seaclouds.yaml (hunk 2 above)
Comment (rosogon) [inline seaclouds.yaml:62]: Any suggestion? Do you mean by prefixing a date, f.e.?

Comment (andreaturli) [inline seaclouds.yaml:62]: Maybe with a permanent link? https://help.github.com/articles/getting-permanent-links-to-files/

**Pull request merged**